config/config_mgmt.py

```diff
@@ -540,8 +545,9 @@ def testRun_Config_Reload_load():
 
     return
 
-def testRun_Delete_Add_Ports():
+def testRun_Delete_Add_Ports_1():
```

Comment:
Add descriptions for 3 test functions?    for example,  Break out from 100G -> 4x25G etc.

Reply:
Changed the Test Code.